package.json

```diff
@@ -1,11 +1,14 @@
 {
   "name": "cwp-starter-theme",
-  "version": "3.0.0",
+  "version": "4.0.0",
   "private": true,
+  "engines": {
+    "node": "^18.x"
+  },
   "scripts": {
-    "watch": "webpack --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js --watch",
-    "dev": "yarn lint && webpack --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
-    "build": "yarn lint && NODE_ENV=production webpack --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
```

Comment:
--hide-modules option is no longer available in the most recent version of laravel-mix, so I just removed it